src/main/plugin/iso19115-3.2018/schematron/schematron-rules-inspire.sch

```diff
@@ -616,8 +616,10 @@
 			<sch:let name="accessConstraints" value="string-join(mco:accessConstraints/*/@codeListValue, ', ')"/>
 			<sch:let name="classification" value="string-join(mco:classification/*/@codeListValue, ', ')"/>
 			<sch:let name="otherConstraints" value="mco:otherConstraints/gco:CharacterString/text()"/>
-			<sch:let name="useLimitation" value="mco:useLimitation/*/text()"/>
-			<sch:let name="useLimitation_count" value="count(mco:useLimitation/*/text())"/>
+			<!-- <sch:let name="useLimitation" value="mco:useLimitation/*/text()"/> -->
+			<sch:let name="useLimitation" value="//mdb:identificationInfo/mri:MD_DataIdentification/mri:resourceConstraints/mco:MD_LegalConstraints/mco:useLimitation/gco:CharacterString/text()"/>
```

Comment:
If you do this, then it does not work for services and the rule also applies to services.

Reply:
Do you mean that my XPATH is too focused wrt the useLimitation tag? if not, could you please explain better my mistake?